ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,570 @@
+		stats: [Stat.StatStrength]
+	},
+	{
+		config: AgilityBuff,
+		picker: IconPicker,
+		stats: [Stat.StatAgility]
+	},
+	{
+		config: IntellectBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatIntellect]
+	},
+	{
+		config: SpiritBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatSpirit]
+	},
+	{
+		config: BattleShoutBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	{
+		config: BlessingOfMightBuff,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	{
+		config: TrueshotAuraBuff,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower, Stat.StatRangedAttackPower]
+		},
+	// {
+	// 	config: AttackPowerPercentBuff,
+	// 	picker: MultiIconPicker,
+	// 	stats: [Stat.StatAttackPower, Stat.StatRangedAttackPower]
+	// },
+	{
+		config: MeleeCritBuff,
+		picker: IconPicker,
+		stats: [Stat.StatMeleeCrit]
+	},
+	{
+		config: SpellIncreaseBuff,
+		picker: IconPicker,
+		stats: [Stat.StatSpellPower]
+	},
+	{
+		config: SpellCritBuff,
+		picker: IconPicker,
+		stats: [Stat.StatSpellCrit]
+	},
+	{
+		config: ResistanceBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatNatureResistance, Stat.StatShadowResistance, Stat.StatFrostResistance]
+		},
+	{
+		config: DefensiveCooldownBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: BlessingOfWisdom,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+	{
+		config: ManaSpringTotem,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+
+	// Misc Buffs
+	{
+		config: Thorns,
+		picker: IconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: RetributionAura,
+		picker: IconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: Innervate,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+	{
+		config: PowerInfusion,
+		picker: IconPicker,
+		stats: [Stat.StatMP5, Stat.StatSpellPower]
+	},
+] as PickerStatOptions[]
+
+export const WORLD_BUFFS_CONFIG = [
+	{
+		config: BoonOfBlackfathom,
+		picker: IconPicker,
+		stats: [
+			Stat.StatMeleeCrit,
+			// TODO: Stat.StatRangedCrit,
+			Stat.StatSpellCrit,
+			Stat.StatAttackPower
+		]
+	},
+	{
+		config: FengusFerocity,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	{
+		config: MoldarsMoxie,
+		picker: IconPicker,
+		stats: [Stat.StatStamina]
+	},
+	{
+		config: RallyingCryOfTheDragonslayer,
+		picker: IconPicker,
+		stats: [
+			Stat.StatMeleeCrit,
+			// TODO: Stat.StatRangedCrit,
+			Stat.StatSpellCrit,
+			Stat.StatAttackPower,
+		]
+	},
+	{
+		config: SaygesDarkFortune,
+		picker: IconEnumPicker,
+		stats: [],
+	},
+	{
+		config: SongflowerSerenade,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: SpiritOfZandalar,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: WarchiefsBlessing,
+		picker: IconPicker,
+		stats: [
+			Stat.StatHealth,
+			Stat.StatMeleeHaste,
+			Stat.StatMP5,
+		]
+	},
+] as PickerStatOptions[];
+
+export const DEBUFFS_CONFIG = [
+	// Standard Debuffs
+	{ 
+		config: MajorArmorDebuff,
+		stats: [Stat.StatArmorPenetration],
```

Comment:
I had changed this to Attack Power in a previous commit, since Armor Penetration doesn't exist as a stat in vanilla (and is something we should probably remove at some point).